tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -119,6 +119,10 @@ class WskBasicUsageTests
             wsk.action.get(fullQualifiedName).stdout should include(s"ok: got action ${packageName}/${actionName}")
     }
 
+    it should "include CLI user agent headers with outbound requests" in {
+        wsk.action.list(verboseFlag = Some(true)).stdout should include regex ("""(?i)User-Agent": \[(\r\n|\r|\n)\s+"OpenWhisk\-CLI/201.*""")
```

Comment:
Little simpler to just check for whitespace instead of the carriage returns and new lines. Also seems like the test will fail in 2020. `"""\bUser-Agent\b": \[\s+"OpenWhisk\-CLI/\d+.*""".r`

Reply:
I see there are other tests that will fail in 2020 as well. We can fix those in a separate PR though.